### PR TITLE
Buttons

### PR DESCRIPTION
**Pull request by moonclash**

### CODE REVIEW

**Comment by moonclash** [comment]
# **Added two button concepts**

buttons

File: src/index.html [hunk 1]
```diff
@@ -28,6 +28,12 @@
       </ul>
     </article>
   </div>
+  <div class="buttons-container">
+    <a href="#" class="btn">get started</a>
+
+    <button class="form-btn">join the ranks</button>
```
Comment (william-abboud) [inline src/index.html]: Okay so before you make a form button just make a primary button. So these styles you've applied at the moment to `.form-btn` should be generic for say a `primary-btn`.
A `button` element should inherit the styles for the primary btn by default without having to add a class. So do something like this:

``` css
button, .primary-btn {
  // Styles for a default button here
}
```

``` html
<button> Register </button>
<a class="primary-btn" href="#"> Enter </a>
```

This is the final effect we are looking for ^

File: src/styles/Base/default.scss [hunk 2]
```diff
@@ -14,5 +14,5 @@ ol {
 
 a {
   text-decoration: none;
-  color: $black;
+  color: $black-500;
```
Comment (william-abboud) [inline src/styles/Base/default.scss:17]: Good spot !

File: src/styles/Elements/buttons.scss [hunk 3]
```diff
@@ -0,0 +1,33 @@
+.btn, .form-btn {
+  text-transform: uppercase;
+}
+
+.btn {
+  background: $yellow-500;
+  border-radius: 18px;
+  box-shadow: 1px 1px 2px $black-400;
+  display: inline-block;
+  padding: 7px 13px;
```
Comment (william-abboud) [inline src/styles/Elements/buttons.scss]: Use `rem` units everywhere except for `border-radius`, `border`, `box-shadow` :)
Also add variation for `large` `small` `medium` buttons aka different paddings ;)

P.S `1rem` = `10px`

File: src/styles/Elements/buttons.scss [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+  transform: scale(1);
+  transition: 0.1s all ease-in-out;
+
+
```
Comment (william-abboud) [inline src/styles/Elements/buttons.scss]: Remove this empty line one is enough :)

File: src/styles/Elements/buttons.scss [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+}
+
+.form-btn {
+  background: none;
+  border: 4px solid $yellow-500;
+  color: $yellow-500;
+  margin-left: 50px;
+  padding: 7px 13px;
+  transition: 0.1s all ease-in-out;
+  &:hover {
```
Comment (william-abboud) [inline src/styles/Elements/buttons.scss:54]: Separate &:hover with a line break

**Pull request merged**